### PR TITLE
format markdnow lint and fix dead link

### PR DESCRIPTION
**Pull request by IcePigZDB**

### CODE REVIEW

**Comment by IcePigZDB** [comment]
hi, @huge-melon, ptal
* I use markdonwlint extension in vscode to check and update some markdown lint.
* Unify to use dash `-` instead mixed use of `_` and `-`.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-| 2022-1-7 | NVM for storage engine | NVM,B+tree,LSM-tree | [doc](./meeting_summary/2022-1-7-基于NVM的数据库存储引擎优化.md) |
-| 2022-02-25 | Database Testing | Database Testing, Optimizer Testing | [doc](./meeting_summary/2022-02-25-数据库系统测试方法介绍.md) |
-| 2022-03-04 | NVM、哈希 | NVM | [doc](./meeting_summary/2022-03-04-面向NVM的数据库哈希索引优化研究.md) |
-| 2022-03-25 | Memory Database | Trie, ART, HOT, Hyperion | [doc](./meeting_summary/2022-03-25-内存数据库简述及索引优化.md) |
+| 2020-09-25 |      Aries           |    Database Recovery, WAL, Fuzzy Checkpoint                  |      [doc](./meeting-summary/2020-09-25-Aries.md)       |
+| 2021-10-29 | Database Transaction | Transaction, ACID, Distributed Transaction, 2PC, Consensus Algorithm |      [doc](./meeting-summary/2021-10-29-数据库事务.md)       |
+| 2021-11-05 |     LSM Tiering      |              LSM、Partition、Leveling、Tiering               | [doc](./meeting-summary/2021-11-05-基于LSM的KV存储写放大优化.md) |
+| 2021-11-12 | Time-Series Database |           IoTDB、Aggregate Query、PISA、Dual-PISA            | [doc](./meeting-summary/2021-11-12-Apache-IoTDB时序数据库.md) |
```
Comment (ehds) [inline README.md:10]: delimiter should be `,`, same with above.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+| 2021-11-19 | SSD | Flash, SSD, Storage System, File System |      [doc](./meeting-summary/2021-11-19-基于SSD的存储系统.md)       |
+| 2021-11-26 | Determinacy database |                      Calvin, PWV, Aria                       |     [doc](./meeting-summary/2021-11-26-确定性数据库.md)      |
```
Comment (ehds) [inline README.md:12]: Capitalize the first letter of the word. `Determinacy database` should be `Determinacy Database`.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+| 2021-12-03 |    Stream system     |            Stream, End to end consistency, Flink             | [doc](./meeting-summary/2021-12-03-流计算端到端一致性概述.md) |
```
Comment (ehds) [inline README.md:13]: `Stream system` should be `Stream System`

File: meeting-summary/2020-09-25-Aries.md [hunk 4]
```diff
@@ -26,9 +26,9 @@
   - 最后,虽然结果是一致的,但是事务的执行需要时间,需要一系列的动作,在这个过程中,系统并没有保证中间状态是一致的。
 - 隔离性要求事务以隔离的方式并发地访问公共资源，彼此不产生干扰。隔离性要求系统小心地控制多个事务去访问公共资源。从一致性结果来看，事务一个一个地迭代执行,总能保证结果是正确的,但是这样性能太低。隔离性的实质是在一致性前提下如何发调度多个事务以取得更高的系统性能。
   - 要对公共资源的访问做隔离，手段无非是两个:一个是时间上隔离，要求事务按一定顺序去访问;另一个是空间上的隔离,对公共资源做多版本。直观上，空间上的隔离更具优势。
--  持久化要求一旦事务正确完成,即使出现系统故障,它的影响也会一直延续。这有两层含义:
-  -  事务完成后,它对公共资源的修改被持久化的记录,后续事务对该资源的访问都受这个事务的影响。
-  -  即使出现系统故障,也不会破坏该事务对资源的改动。
+- 持久化要求一旦事务正确完成,即使出现系统故障,它的影响也会一直延续。这有两层含义:
```
Comment (ehds) [inline meeting-summary/2020-09-25-Aries.md:29]: This document contains many mathematical formulas which not be parsed by github markdown parser correctly.

**Review by ehds — CHANGES_REQUESTED** [review]

File: meeting-summary/2020-09-25-Aries.md (hunk 4 above)
Comment (IcePigZDB) [reply meeting-summary/2020-09-25-Aries.md:29]: I am going to use github official latex render [service ](https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b) on this point.

**Comment by ehds** [comment]
Nice PR! LGTM.

**Review by ehds — APPROVED** [review]

**Comment by IcePigZDB** [comment]
@ehds, use github render service in areis, ptal.

**Comment by ehds** [comment]
> @ehds, use github render service in areis, ptal.

LGTM, Is there any tools to complete the replacement automatically?

**Comment by IcePigZDB** [comment]
> > @ehds, use github render service in areis, ptal.
> 
> LGTM, is there any tools to complete the replacement automatically?

not yet

**Comment by ehds** [comment]
> > > @ehds, use github render service in areis, ptal.
> > 
> > 
> > LGTM, is there any tools to complete the replacement automatically?
> 
> not yet

A simple solution: 
1. find mathematical formula by regular expression.
2. replace matched value.

**Comment by IcePigZDB** [comment]
> > > > @ehds, use github render service in areis, ptal.
> > > 
> > > 
> > > LGTM, is there any tools to complete the replacement automatically?
> > 
> > 
> > not yet
> 
> A simple solution:
> 
> 1. find mathematical formula by regular expression.
> 2. replace matched value.

update in #27

**Pull request merged**